bibliopixel/layout/layout.py

```diff
@@ -7,18 +7,19 @@
 class Layout(object):
 
     def __init__(self, drivers, threadedUpdate, brightness,
-                 maker=data_maker.MAKER, **kwds):
+                 maker=data_maker.MAKER, pixel_size=3, **kwds):
```

Comment:
I personally hate this name. It's not the size of the pixel. It's the number of color channels.